app/interactors/delete_user.rb

```diff
@@ -0,0 +1,19 @@
+class DeleteUser
+  include Interactor
+
+  def call
+    validate_input
+    execute
+  end
+
+  private
+
+  def validate_input
+    context.fail!(error: "invalid input") unless context.id
+  end
+
+  def execute
+    context.user = User.find(context.id)
+    context.fail!(error: "user not deleted") unless context.user.destroy
```

Comment:
@Enriikke I didn't user a `validate_output` here, just this `fail!`. On interactors where you're not actually returning anything (like a user) would you do the same or structure this differently?